test/UnitTest/Microsoft.Test.AspNet.Common/AspNet/TestControllers.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web.Http;
+using System.Web.Http.Results;
+using Microsoft.AspNet.OData;
+using Microsoft.AspNet.OData.Results;
+
+namespace Microsoft.Test.AspNet.OData
+{
+    /// <summary>
+    /// NonODataController is an abstracted ASP.NET [Core] controller.
+    /// </summary>
+    public class TestNonODataController : ApiController
```

Comment:
Since you are splitting files, maybe make one for TestNonODataController and one for TestODataController ?

Reply:
Great idea. Copy that.

Reply:
@robward-ms Done!